tests/data/power_flow/dummy-test/sym_output.json

```diff
@@ -60,15 +60,15 @@
       {
         "id": 7,
         "i": 24.24871130596428,
-        "p": 352799.85244129569,
+        "p": 352800.0,
```

Comment:
Why?

Reply:
it's a precision issue. not sure there's much we can do to improve it here

Reply:
For some reason I didn't realize they are basically the same value. Oops... Resolving.